stories/Molecules/FooterNavigation/FooterIcons/footerIcons.scss

```diff
@@ -21,23 +22,23 @@
       width: 20px;
 
       &.facebook {
-        background: url(#{$img-path-Icon}/facebook.svg) no-repeat center bottom;
+        background: url(#{$img-path-Icon}/facebook-white.svg) no-repeat center bottom;
       }
 
       &.instragram {
-        background: url(#{$img-path-Icon}/instagram.svg) no-repeat center bottom;
+        background: url(#{$img-path-Icon}/instagram-white.svg) no-repeat center bottom;
       }
 
       &.linkedin {
-        background: url(#{$img-path-Icon}/linkedIn.svg) no-repeat center bottom;
+        background: url(#{$img-path-Icon}/linkedin-white.svg) no-repeat center bottom;
       }
 
       &.twitter {
-        background: url(#{$img-path-Icon}/twitter.svg) no-repeat center bottom;
+        background: url(#{$img-path-Icon}/twitter-white.svg) no-repeat center bottom;
       }
 
       &.youtube {
-        background: url(#{$img-path-Icon}/youtube.svg) no-repeat center bottom;
+        background: url(#{$img-path-Icon}/youtube-white.svg) no-repeat center bottom;
```

Comment:
@Shubham-Sharma-77 Can you write this as:
```
$socials: ("facebook", "instagram", "linkedin", "twitter", "youtube");
@each $name in $socials {
  &.#{$name}{
    background: url(#{$img-path-Icon}/#{$name}-white.svg) no-repeat center bottom;
  }
}
```
This will compile to:
```
a.facebook {
  background: url(../../facebook-white.svg) no-repeat center bottom;
}
a.instagram {
  background: url(../../instagram-white.svg) no-repeat center bottom;
}
a.linkedin {
  background: url(../../linkedin-white.svg) no-repeat center bottom;
}
a.twitter {
  background: url(../../twitter-white.svg) no-repeat center bottom;
}
a.youtube {
  background: url(../../youtube-white.svg) no-repeat center bottom;
}

```